desc/vmec_utils.py

```diff
@@ -233,3 +234,28 @@ def zernike_to_fourier(x_lmn, basis, rho):
             x_mn[:, k] = np.matmul(A, x_lmn[idx])
 
     return m, n, x_mn
+
+
+"""
+# an alternative version of ptolemy_identity_fwd
```

Comment:
@ddudt can this be deleted?

Reply:
The current `vmec_utils` functions are tailored specifically to their usage in the `VMECIO` class. I created this alternative version of the function for a custom application that was not handled well by the original implementation. It looks like the reason could have been that the DESC basis had a higher resolution than the VMEC basis. 
So yes, this could be deleted. But long-term it would be good to generalize this and the other util functions.